operate/schema/src/main/java/io/camunda/operate/entities/dmn/DecisionInstanceEntity.java

```diff
@@ -22,7 +22,8 @@ public class DecisionInstanceEntity extends OperateZeebeEntity<DecisionInstanceE
   private Integer executionIndex;
   private DecisionInstanceState state;
   private OffsetDateTime evaluationDate;
-  private String evaluationFailure;
+  @Deprecated private String evaluationFailure;
```

Comment:
Sorry to put a new comment here that wasn't on the original PR, but I am thinking it might be helpful to use the `since` parameter in the annotation. This way, it builds up a bit of a change log over time. So here, it would be:
`@Deprecated(since = "8.7.x")` (I can't remember which patch version we are on).

Wdyt?